main.py

```diff
@@ -0,0 +1,176 @@
+import sys
+from csv import reader
+import datetime
+from math import ceil
+import os
+from statistics import mean
+
+
+def main(argv):
+    months = [" ", "Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"]
+
+    length = len(sys.argv)
+    option = sys.argv[length - 2]
+    date = sys.argv[length - 1]
+    print(date)
+    if option == "-e":
+
```

Comment:
Any particular reason for leaving this blank line? Relevant blocks of code should be together to improve readability.